subsys/bluetooth/host/l2cap.c

```diff
@@ -1707,7 +1703,8 @@ void bt_l2cap_recv(struct bt_conn *conn, struct net_buf *buf)
 		return;
 	}
 
-	l2cap_chan_recv_queue(chan, buf);
+	l2cap_chan_recv(chan, buf);
+	net_buf_unref(buf);
```

Comment:
This change confuses me somewhat. Where was this done previously for the static channels?

Reply:
In the k_work callback that `l2cap_chan_recv_queue` schedules, i.e. `l2cap_rx_process()`, which then eventually called `l2cap_chan_recv()`